auth-table-based-service/src/main/java/io/stargate/auth/table/AuthTableBasedService.java

```diff
@@ -157,12 +154,12 @@ private String queryUsername(String key) throws ExecutionException, InterruptedE
             .where("role", WhereCondition.Predicate.Eq, key)
             .execute();
 
-    if (resultSet.isEmpty()) {
+    if (resultSet.hasNoMoreFetchedRows()) {
       throw new RuntimeException(String.format("Provided username %s is incorrect", key));
```

Comment:
This is not related this PR again, but I believe in general it is not advisable to disclose information about user name existence if the credentials have not been verified... It would be preferable to report `Provided username %s and/or password are incorrect`, WDYT?

Reply:
Agreed, and that one is simple enough that I'll sneak it in.